src/Items/ItemFishingRod.h

```diff
@@ -100,26 +100,37 @@ class cItemFishingRodHandler :
 			}
 			else if (FloaterInfo.CanPickup())
 			{
+				uint LotSLevel = a_Player->GetEquippedItem().m_Enchantments.GetLevel(cEnchantments::enchLuckOfTheSea);
```

Comment:
This breaks the build on windows. It would be better to use a UInt32

Reply:
Done